app/controllers/dashboards_controller.rb

```diff
@@ -2,5 +2,15 @@ class DashboardsController < ApplicationController
   before_filter :require_login
 
   def show
+    unless current_person.kind_of? String
+      @to_apply_applications = application_search('to_apply')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.